src/Compilers/Core/Portable/PEWriter/MetadataWriter.PortablePdb.cs

```diff
@@ -990,5 +991,28 @@ private void EmbedMetadataReferenceInformation(CommonPEModuleBuilder module)
                 kind: _debugMetadataOpt.GetOrAddGuid(PortableCustomDebugInfoKinds.CompilationMetadataReferences),
                 value: _debugMetadataOpt.GetOrAddBlob(builder));
         }
+
+        private void TypeDocumentInformation(CommonPEModuleBuilder module)
+        {
+            DocumentHandle rowid;
+            var document = module.GetTypeDocument(Context);
```

Comment:
>document [](http://example.com/codeflow?start=16&length=8)

The method returns a list, so the variable name should reflect that and be plurar.